spec/index.html

```diff
@@ -9537,6 +9538,11 @@ <h4>Aggregate Algebra</h4>
             <p>The Flatten(S) function takes a sequence of lists, S = [(L<sub>1</sub>, L<sub>2</sub>,
               ...), ...], and returns the list ( x | L in S and x in L ).</p>
           </div>
+          <p>Card is a function that returns the cardinality of a sequence or a list of elements (which may be solution mappings or other types of elements, depending on the context).
+          <div class="defn">
+            <p><b>Definition: <span id="defn_Card">Card</span></b></p>
+            <p>Given a sequence or a list |L|, Card(|L|) is the cardinality of |L|.</p>
```

Comment:
Should the list be named just `L`, with `|L|` being syntatic sugar for `Card(L)`?

```suggestion
            <p>Given a sequence or a list L, Card(L) is the cardinality of L.</p>
```

Reply:
That's [respec shorthand markup for a variable](https://respec.org/docs/#variable-shorthands).

Reply:
Ah, TIL, thanks for letting me know!